_search-plugins/search-relevance/index.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: default
+title: Search relevance
+nav_order: 55
+has_children: false
+has_toc: false
+redirect_from:
+  - /search-plugins/search-relevance/
+---
+
+# Search relevance
+
+Search relevance is an experimental feature. Therefore, we do not recommend the use of search relevance in a production environment. For updates on the progress of search relevance, or if you want to leave feedback that could help improve the feature, join the [discussion on the OpenSearch forum]((https://forum.opensearch.org/t/feedback-experimental-feature-compare-search-results/11331).    
```

Comment:
Will remove.

Reply:
@kolchfa-aws: Since this is boilerplate text, let's keep it in here.

@lauralexis: For context, our general rule with all experimental features to use them with caution, which includes using `read_only` features for production datasets. For consistency with other experimental features, I say we keep the warning in, then remove once the feature is no longer behind a feature flag.

Reply:
I think it makes sense to err on the side of caution, however, the user might question why we are recommending it shouldn't be used with production data sets, and I can't think of a reason. 

@macrakis Can you think of a reason this tool shouldn't be used with production data sets?